docs/practitioner/energy-efficiency.mdx

```diff
@@ -124,12 +124,12 @@ Servers are usually not configured for aggressive or even minimal power saving.
     {
       question: "What is clean energy?",
       answers: [
-        { text: "Energy that comes from renewable sources", isCorrect: true },
+        { text: "Energy that comes from renewable sources", isCorrect: false },
         {
           text: "Energy that doesn’t produce carbon emissions",
-          isCorrect: true,
```

Comment:
this should be the answer (https://learn.greensoftware.foundation/practitioner/energy-efficiency#low-carbon-sources-of-energy)